### PR TITLE
Add 'web3 scripts' (to FE context menu)

### PR DESCRIPTION
**Pull request by yann300**
add scripts for:
 - etherscan
 - contract deployer (create2)

### CODE REVIEW

**Comment by yann300** [comment]
some more insight about this:
Another solution would be to add these helper scripts directly to the templates.

File: apps/remix-ide/src/app/tabs/locales/en/filePanel.json [hunk 1]
```diff
@@ -29,6 +29,8 @@
   "filePanel.tssoltestghaction": "Mocha Chai Test Workflow",
   "filePanel.workspace.slitherghaction": "Adds a preset yml file to run slither analysis on github actions CI",
   "filePanel.slitherghaction": "Slither Workflow",
+  "filePanel.workspace.helperscripts": "Adds convenient scripts to the 'scripts' directory",
```
Comment (ioedeveloper) [inline apps/remix-ide/src/app/tabs/locales/en/filePanel.json:32]: Can you also include the translation for it?

Comment (yann300) [reply apps/remix-ide/src/app/tabs/locales/en/filePanel.json:32]: yes I've added the translation label.
cc @ryestew

**Comment by yann300** [comment]
after discussion, we should have sub menus for this.

**Review by joeizang — APPROVED** [review]
LGTM

**Pull request merged**